bot/exts/moderation/modlog.py

```diff
@@ -629,13 +631,15 @@ async def log_uncached_deleted_message(self, event: discord.RawMessageDeleteEven
             response = (
                 f"**Channel:** {channel.category}/#{channel.name} (`{channel.id}`)\n"
                 f"**Message ID:** `{event.message_id}`\n"
+                f"**Sent at:** `{format_dt(snowflake_time(event.message_id), style='f')}`\n"
```

Comment:
Is there a reason this one uses lowercase `f` formatting instead of uppercase? There does seem to be a difference https://discord.com/developers/docs/reference#message-formatting-timestamp-styles.

This also shouldn't be in backticks

Reply:
@MaskDuck Could you please make sure it's the format that doesn't include the day of the week?

March 31, 2022 [time]